jsonnet/kube-prometheus/addons/pyrra.libsonnet

```diff
@@ -103,7 +107,10 @@
             spec: {
               containers: [c],
               // serviceAccountName: $.serviceAccount.metadata.name,
-              nodeSelector: { 'kubernetes.io/os': 'linux' },
+              nodeSelector: {
+                'kubernetes.io/os': 'linux',
+                'kubernetes.io/arch': 'amd64',
```

Comment:
This can be removed when pyrra publishes arm64 images (WIP: https://github.com/pyrra-dev/pyrra/pull/249)